frontend/src/routes/testing/+page.svelte

```diff
@@ -1,160 +1,432 @@
+          <div class="card w-full">
+            <figure>
+              <figcaption class="p-4 card-body flex flex-col">
+                <h2 class="card-title mb-2">Time-Bound</h2>
+                <div class="flex flex-col md:flex-row w-full">
+                  <div
+                    class="border rounded-xl w-full py-2 px-3 leading-tight focus:outline-none focus:shadow-outline flex-grow mb-2 md:mb-0"
+                  >
+                    AchievAIm's Time-Bound suggestion
+                  </div>
+                </div>
+              </figcaption>
+            </figure>
+          </div>
+
+          <div class="mt-3 flex flex-col items-center">
+            <div class="card w-full">
+              <figure>
+                <figcaption class="p-4 card-body flex flex-col items-center">
+                  {#if goal.days_of_week}
+                    <div class="flex justify-between items-center w-full">
+                      <h2 class="card-title mb-2">Days</h2>
+                    </div>
+                    <DaysOfWeekSelector daysOfWeek={goal.days_of_week} />
+                  {/if}
                 </figcaption>
               </figure>
             </div>
-            <!-- Relevant card -->
-            <div class="card">
+          </div>
+
+          <div class="card w-full">
+            <figure>
+              <figcaption class="p-4 card-body flex flex-row items-center">
+                <h2 class="card-title mb-2">Date</h2>
+                <div class="flex-grow flex items-center relative">
+                  <input
+                    class="shadow appearance-none border rounded w-full py-2 px-3 leading-tight focus:outline-none focus:shadow-outline"
+                    id="goal-date"
+                    type="date"
+                    bind:value={goal.date_for_achievement}
+                    aria-describedby="date-description"
+                    readonly
+                  />
+                </div>
+              </figcaption>
+            </figure>
+          </div>
+
+          <div class="card w-full">
+            <figure>
+              <figcaption class="p-4 card-body flex flex-row items-center">
+                <h2 class="card-title mb-2">Time</h2>
+                <div class="flex-grow flex items-center">
+                  <input
+                    class="shadow appearance-none border rounded w-full py-2 px-3 leading-tight focus:outline-none focus:shadow-outline"
+                    id="goal-time"
+                    type="time"
+                    bind:value={goal.time_of_day}
+                    aria-describedby="time-description"
+                    readonly
+                  />
+                </div>
+              </figcaption>
+            </figure>
+          </div>
+        </div>
+        <!-- Card 2 -->
+        <div class="w-full bg-white rounded-xl p-4 shadow-md mb-4">
+          <!-- Content for Card 2 -->
+          <p>Card 2</p>
+        </div>
```

Comment:
Is this supposed to be here?

Reply:
https://github.com/xyvora/achievaim/pull/274#discussion_r1285136122 → Yes and https://github.com/xyvora/achievaim/pull/274#discussion_r1285136257 → Yes.

I will create a pull request that will allow you to view the updated and final layout of the card page on the collapsecard branch. Before making any changes to the current home/main/display goal page, I wanted to work with you to ensure that I don't accidentally delete any logic or change anything you have already implemented. That's why I used format, design, and testing pages. To see the new drop-down collapsible card I was working on, please check out the testing and testingcomponent pages. The testing page shows an example layout with two "active" collapsible cards and one "completed" card, while the testingcomponent page displays a single collapsible card.